contracts/AxelarGateway.sol

```diff
@@ -324,13 +324,25 @@ contract AxelarGateway is IAxelarGateway, AdminMultisigBase {
                 continue; /* Ignore if unknown command received */
             }
 
+            uint256 availableGas = gasleft();
+            if (availableGas < 100000) return;
+
             // Prevent a re-entrancy from executing this command before it can be marked as successful.
             _setCommandExecuted(commandId, true);
             // solhint-disable-next-line avoid-low-level-calls
-            (bool success, ) = address(this).call(abi.encodeWithSelector(commandSelector, params[i], commandId));
+            (bool success, ) = address(this).call{ gas: availableGas - 10000 }(
```

Comment:
```suggestion
            (bool success, ) = address(this).call{ gas: availableGas - 100000 }(
```
I prefer keeping a higher budget to be safe. doesn't require us to pay more gas, just decreases the max batching possibility by a bit.

Reply:
Yeah that makes sense. I will close this PR in favor of the actual fix